compiler/optimizer/CopyPropagation.cpp

```diff
@@ -1230,7 +1230,7 @@ void TR_CopyPropagation::replaceCopySymbolReferenceByOriginalIn(TR::SymbolRefere
                if (origNode->getNumChildren() == 2)
                   {
                   twoChildrenCase = true;
-                  TR_ASSERT(0, "Only in WCode we can add extra children\n");
+                  TR_ASSERT_FATAL(0, "Only in WCode we can add extra children\n");
```

Comment:
The fatal assert message should be re-written to be "Cannot add extra children".  The WCode message is no longer relevant.

Reply:
Fixed. Maybe the code handling the `twoChildrenCase` should be removed in future?